packages/11ty/.gitignore

```
@@ -1,6 +1,9 @@
 # Eleventy build artefacts
 _site/
 
+# Generated IIIF image files
+content/_assets/images/_iiif/
```

What do you thinkg of removing `/images/` from the output path for the generated images?
```suggestion
content/_assets/_iiif/
```